README.md

```diff
@@ -1,14 +1,12 @@
-# bookstore-redux
-Keeps track of books that you have read using React and Redux.
-
-![Microverse](https://img.shields.io/badge/Microverse2.0-blueviolet) ![app](https://img.shields.io/badge/Myapp-blue
-React project has been made with HTML5, CSS3, JS, JSX and Webpack, following the guidelines of 
-![Microverse](https://img.shields.io/badge/Microverse2.0-blueviolet)[🔗](https://www.microverse.org/) for best practices.
+![Microverse](https://img.shields.io/badge/Microverse2.0-blueviolet) ![app](https://img.shields.io/badge/Myapp-blue)
 ---
```

Comment:
**Changes Required**
- [ ] The mobile and desktop images are broken and the link to the `live demos` are broken as well. The README acts as a documentation and you want to keep it updated as much as possible. Let us fix these changes accordingly.

Reply:
no, neither mobile nor desktop is a requirement, I only put it there because this project is of many milestones, mobile version isn't named in the whole project, I left it there in case I get extra time to build one, and it's really not needed any preview at this point of the project, this is just the setup, it's just to check if I'm doing the right work-flow when I approve I have to build 3 more parts of the project. And in any case, this should be a suggested change, not a requirement since Microverse didn't ask for it at this stage, neither deployment here at the 1st milestone. And it's not enough to hold me back from approval.

Reply:
- [ ] I understood where your coming from. The README file might not be a `project requirement` but it is always under `general requirements`. As I said, it's the first thing anyone needing to review or contribute to your project will go to in order to understand what your project is about. Please always keep it professional according to the Microverse General Requirements guidelines.

Reply:
however I fixed it  4 hours ago.